store/dataStore.test.ts

```diff
@@ -3,6 +3,7 @@ import { useDataStore } from './dataStore';
 
 // test values
 const userData = {
+  documentId: '123',
   userId: '123',
   userEmail: 'test@email.com',
   leagues: ['123456'],
```

Comment:
this is incorrect.  should be: documentId, id, email, leagues.  not userId and userEmail.

Reply:
Updated

Reply:
@vmaineng look like at the spelling for id?  looks like you have it as `idd`

Reply:
@vmaineng I have another question.  Why did you update the datastore test when the goal is to test the /league/all page?  I just want to make sure we are not updating files that don't need to be updated. (I'm guilty of this too hehe 🤣)

Reply:
@chris-nowicki initially, I thought it wasn't supposed to be changed, but I do recall putting in an 'x' in front of the first test in `describe('User Test',` so I had to correct it. Please review this test where I added in the documentId

Reply:
got it!